src/synchronization/stackImagePositionSynchronizer.js

```diff
@@ -15,17 +15,40 @@ export default function (synchronizer, sourceElement, targetElement) {
   const cornerstone = external.cornerstone;
   const sourceImage = cornerstone.getEnabledElement(sourceElement).image;
   const sourceImagePlane = cornerstone.metaData.get('imagePlaneModule', sourceImage.imageId);
-  const sourceImagePosition = convertToVector3(sourceImagePlane.imagePositionPatient);
+  let sourceImagePosition;
+
+  if (sourceImagePlane !== undefined) {
+    sourceImagePosition = convertToVector3(sourceImagePlane.imagePositionPatient);
+  }
+
+  if (sourceImagePosition === undefined) {
+    // Console.log('No position found for image ' + sourceImage.imageId);
+
+    return;
+  }
 
   const stackToolDataSource = getToolState(targetElement, 'stack');
   const stackData = stackToolDataSource.data[0];
 
   let minDistance = Number.MAX_VALUE;
   let newImageIdIndex = -1;
+  let nbComparedPositions = 0;
```

Comment:
I guess this is a remnant of testing? If you remove nbComparedPositions I think we can merge this PR